DEPENDS.txt

```diff
@@ -1,7 +1,7 @@
 Build Requirements
 ------------------
 * `Python >= 2.6 <http://python.org>`__
-* `Numpy >= 1.6.1 <http://numpy.scipy.org/>`__
+* `Numpy >= 1.7.2 <http://numpy.scipy.org/>`__
```

Comment:
It's about time for that anyway. It solves precision issues in another PR.
On Jan 31, 2016 10:58, "Johannes Schönberger" <notifications@github.com>
wrote:

> In DEPENDS.txt
> <https://github.com/scikit-image/scikit-image/pull/1827#discussion_r51365235>
> :
>
> > @@ -1,7 +1,7 @@
> >  Build Requirements
> >  ------------------
> >  * `Python >= 2.6 <http://python.org>`__
> > -* `Numpy >= 1.6.1 <http://numpy.scipy.org/>`__
> > +* `Numpy >= 1.7.2 <http://numpy.scipy.org/>`__
>
> We don't need to raise to 1.7.x then?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-image/scikit-image/pull/1827/files#r51365235>.
>